db/migrations/20260821_0052_marketplace_guest_final_hardening.sql

```
@@ -0,0 +1,56 @@
+-- Final database hardening for marketplace guest invitations.
+--
+-- Production forward deployment order: apply 0049, 0050, 0051, then this
+-- migration before deploying the final marketplace guest application changes.
+-- Committing this file does not apply it to Production.
+
+begin;
+
+-- Match the normalized recipient expression used by suppression checks and
+-- opt-out queries so the lookup remains indexable for whitespace/case variants.
+drop index if exists marketplace_quote_invitations_recipient_idx;
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)